### PR TITLE
contracts-bedrock: make spacer variable non-public

### PR DESCRIPTION
**Pull request by maurelian**
**Description**

A spacer variable was accidentally labelled public. There is no need for it to be part of the ABI. 

This removes it.

### CODE REVIEW

File: packages/contracts-bedrock/src/L1/OptimismPortal.sol [hunk 1]
```diff
@@ -61,7 +61,7 @@ contract OptimismPortal is Initializable, ResourceMetering, ISemver {
     /// @custom:legacy
     /// @custom:spacer paused
     /// @notice Spacer for backwards compatibility.
-    bool public spacer_53_0_1;
+    bool spacer_53_0_1;
```
Comment (mds1) [inline packages/contracts-bedrock/src/L1/OptimismPortal.sol]: Nit, looks like we're explicit about visibility elsewhere which I agree with
```suggestion
    bool private spacer_53_0_1;
```

Comment (tynes) [reply packages/contracts-bedrock/src/L1/OptimismPortal.sol]: agree with making it explicit

File: packages/contracts-bedrock/src/L1/OptimismPortal.sol [hunk 2]
```diff
@@ -61,7 +61,7 @@ contract OptimismPortal is Initializable, ResourceMetering, ISemver {
     /// @custom:legacy
     /// @custom:spacer paused
     /// @notice Spacer for backwards compatibility.
-    bool public spacer_53_0_1;
+    bool private spacer_53_0_1;
```
Comment (tynes) [inline packages/contracts-bedrock/src/L1/OptimismPortal.sol:64]: Needs an update to semver lock

Comment (maurelian) [reply packages/contracts-bedrock/src/L1/OptimismPortal.sol:64]: incoming :)

**Review by tynes — APPROVED** [review]

**Pull request merged**